### PR TITLE
Grid imported features

### PR DESCRIPTION
**Pull request by feenster**
@hunt3ri @LindaAlblas please can you review the server and client stuff separately

Linda, if you want to test you will need to run the server locally.  tmAPI in taskingmanager.config.json points to localhost:5000 in this branch.

### CODE REVIEW

File: client/taskingmanager.config.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "development": {
     "EnvironmentConfig":{
-      "tmAPI": "http://tasking-manager-staging.eu-west-1.elasticbeanstalk.com/api/v1",
```
Comment (LindaAlblas) [inline client/taskingmanager.config.json]: Could you put this back to the staging API please?

File: server/services/grid_service.py [hunk 2]
```diff
@@ -57,3 +73,71 @@ def _update_feature(clip_to_aoi: bool, feature: dict, new_shape):
             feature['properties']['splittable'] = False
         return feature
 
+    @staticmethod
+    def _parse_geojson(input: str)->list:
```
Comment (hunt3ri) [inline server/services/grid_service.py]: Suggest renaming this to:

_to_shapely_features(geojson_collection: str) -> list

File: server/services/grid_service.py [hunk 3]
```diff
@@ -57,3 +73,71 @@ def _update_feature(clip_to_aoi: bool, feature: dict, new_shape):
...
+        """
+        Parses the input geojson and returns a list of geojson.Fearure objects with their geometries adapted to shapely geometries
+        :param input: string of geojson
+        :return: list of geojson.Fearure objects with their geometries adapted to shapely geometries
+        """
+        collection = geojson.loads(input,
```
Comment (hunt3ri) [inline server/services/grid_service.py]: Put on one line

File: server/services/grid_service.py [hunk 4]
```diff
@@ -57,3 +73,71 @@ def _update_feature(clip_to_aoi: bool, feature: dict, new_shape):
...
+                                   object_hook=geojson.GeoJSON.to_instance)
+
+        if not hasattr(collection, "features") or \
```
Comment (hunt3ri) [inline server/services/grid_service.py]: Put if statement on one line

File: server/services/grid_service.py [hunk 5]
```diff
@@ -57,3 +73,71 @@ def _update_feature(clip_to_aoi: bool, feature: dict, new_shape):
...
+
+        geometry = MultiPolygon(rings)
+
+        # Downsample 3D -> 2D
+        wkt2d = geometry.wkt
+        geom2d = shapely.wkt.loads(wkt2d)
+
+        return geom2d
+
+    def _dissolve(geoms: MultiPolygon)->MultiPolygon:
```
Comment (hunt3ri) [inline server/services/grid_service.py]: This should be static, right?

File: server/services/grid_service.py (hunk 2 above)
Comment (feenster) [inline server/services/grid_service.py]: Done

**Comment by feenster** [comment]
Yes, all tests passing, and config comment from Linda handled.  Thanks.

**Pull request merged**